# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/svg/reftest.list
# Commit: beec388d4b0e
# Full Hash: beec388d4b0e6a50ac243ed86c0e3f1f48cbe744
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-06-25 09:36:02
# Regressor Bug: 1896503
# File Overlap Count: 1
# Description:
#   Bug 1896503 - Implement FilterInstance code to send SVG filter graph to WebRender r=mstange,gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D174208
# ==============================================================================

diff -r 2147e0051e23 -r beec388d4b0e layout/reftests/svg/reftest.list
--- a/layout/reftests/svg/reftest.list	Mon Jun 24 23:40:38 2024 +0000
+++ b/layout/reftests/svg/reftest.list	Mon Jun 24 23:45:05 2024 +0000
@@ -61,7 +61,10 @@
 == active-transform-blend-mode.html active-transform-blend-mode-ref.html
 == active-clip-and-mask.html active-clip-and-mask-ref.html
 
-fuzzy(0-11,0-7155) == blur-inside-clipPath.svg blur-inside-clipPath-ref.svg
+# this test exists to detect if clip paths are erroneously cropped by the
+# unfiltered size of the primitive, which was bug 1459890, so the blur fuzz
+# doesn't matter, as if the error occurs it causes a much more drastic change
+fuzzy(0-20,0-9840) == blur-inside-clipPath.svg blur-inside-clipPath-ref.svg
 
 == border-radius-01.html pass.svg
 == mask-image-filter-transform.html mask-image-filter-transform-ref.html
@@ -177,7 +180,7 @@
 == dynamic-stroke-width-01.svg pass.svg
 == dynamic-switch-01.svg pass.svg
 == dynamic-text-01.svg dynamic-text-01-ref.svg
-fuzzy-if(winWidget,0-3,0-12739) == dynamic-text-02.svg dynamic-text-02-ref.svg # bug 776038 for Win7, Win8
+fuzzy-if(winWidget,0-4,0-12739) fuzzy-if(gfxSVGFEGaussianBlur,0-7,0-1000000) == dynamic-text-02.svg dynamic-text-02-ref.svg # bug 776038 for Win7, Win8
 fuzzy-if(winWidget,0-2,0-10539) == dynamic-text-03.svg dynamic-text-03-ref.svg # bug 776038 for Win7
 == dynamic-text-04.svg dynamic-text-04-ref.svg
 == dynamic-text-05.svg pass.svg
@@ -471,7 +474,7 @@
 == suspend-08.svg pass.svg
 
 == svg-effects-area-unzoomed.xhtml svg-effects-area-unzoomed-ref.xhtml
-== svg-effects-area-zoomed-in.xhtml svg-effects-area-zoomed-in-ref.xhtml
+fuzzy-if(geckoview&&!emulator&&gfxSVGFEComponentTransfer,0-4,0-305) == svg-effects-area-zoomed-in.xhtml svg-effects-area-zoomed-in-ref.xhtml
 == svg-effects-area-zoomed-out.xhtml svg-effects-area-zoomed-out-ref.xhtml
 == svg-transform-01.svg pass.svg
 == svg-transform-02.svg pass.svg